tests/test_exceptions.py

```diff
@@ -52,7 +52,7 @@ def test_no_inputs():
 build_json_dir=/nonexistent/path/
 
 [default]
-openshift_uri=https://172.0.0.1:8443/
+openshift_url=https://172.0.0.1:8443/
```

Comment:
Did we change compatibility of config files?

Reply:
_uri VS _url

Reply:
it's the only way to avoid a deprecated value warning that is preventing Travis from passing successfully.